r-pkg/DESCRIPTION

```diff
@@ -4,11 +4,11 @@ Title: Get Data Frame Representations of 'Elasticsearch' Results
 Version: 0.4.0.9999
 Authors@R: c(
     person("James", "Lamb", email = "jaylamb20@gmail.com", role = c("aut", "cre")),
-    person("Nick", "Paras", email = "nick.paras@uptake.com", role = c("aut")),
-    person("Austin", "Dickey", email = "austin.dickey@uptake.com", role = c("aut")),
+    person("Nick", "Paras", role = c("aut")),
+    person("Austin", "Dickey", role = c("aut")),
```

Comment:
@austin3dickey let me know if you want a different email added here! I just assumed you wouldn't, but happy to add one.

Reply:
Yeah, probably not. I still follow Issues closely if people want to reach out 🙂

Reply:
alright I'll leave it like this, thanks